src/components/Navbar.js

```diff
@@ -1,7 +1,11 @@
+import { useState } from "react";
 import { Link } from "react-router-dom";
 import { NavbarData } from "../data";
+
 export const Navbar = () => {
   const { categories, navbarActions } = NavbarData;
```

Comment:
You are not getting the categories from the backend right?

Reply:
No not yet.